zcash_client_sqlite/src/wallet.rs

```diff
@@ -154,7 +155,12 @@ fn parse_account_source(
                 )
             })?,
         }),
-        (1, None, None) => Ok(AccountSource::Imported),
+        (1, None, None) => Ok(AccountSource::Imported {
+            account_type: AccountType::ViewOnly,
+        }),
+        (2, None, None) => Ok(AccountSource::Imported {
+            account_type: AccountType::Spending,
+        }),
         (0, None, None) | (1, Some(_), Some(_)) => Err(SqliteClientError::CorruptedData(
```

Comment:
```suggestion
        (0, None, None) | (1, Some(_), Some(_)) | (2, Some(_), Some(_)) => Err(SqliteClientError::CorruptedData(
```

Reply:
Ah, crud, this actually means a database constraint (that I didn't really want in the first place) has to change. Grrr.